bridge/server/app.js

```diff
@@ -27,6 +31,52 @@ if(!integrationsPageLink) {
   integrationsPageLink = "https://get.keptn.sh/integrations.html";
 }
 
+
+try {
+  console.log("Installing default Look-and-Feel");
+
+  let destDir = path.join(__dirname, '../dist/assets/branding');
+  let srcDir = path.join(__dirname, '../client/assets/default-branding');
+
+  let brandingFiles = ["app-config.json", "logo.png", "logo_inverted.png"];
+  brandingFiles.forEach((file) => {
+    if(!fs.existsSync(destDir)) {
```

Comment:
Move this check outside the loop or change the path to `path.join(srcDir, file)`.
I would also move `destDir` and the check if it exists one layer up and remove the declaration and check during the custom look and feel installation

Because the variables are not overwritten, I would change it to `const`

Reply:
You are right, the check if the directory exists can be done once, instead of inside the loop.